README.md

```diff
@@ -171,6 +165,23 @@ PayPal used to support authentication via a modal iframe, but authentication now
 
 PopupBridge solves this problem by allowing [`braintree-web`](https://github.com/braintree/braintree-web) to open the PayPal popup from a secure mini-browser.
 
+Using PayPal in a WebView
+-------------------------
+
+WebView-based checkout flows can accept PayPal with PopupBridge the Braintree JS SDK. For the authentication flow, PayPal requires a popup window—which can be simulated with PopupBridge.
```

Comment:
"PopupBridge the Braintree JS SDK" => "PopupBridge and the [Braintree JS SDK](link)"